chart/flux/templates/deployment.yaml

```diff
@@ -175,7 +175,7 @@ spec:
           {{- end }}
           args:
           {{- if not .Values.clusterRole.create }}
-          - --k8s-allow-namespace={{ .Release.Namespace }}
+          - --k8s-allow-namespace={{ join "," (append .Values.allowedNamespaces .Release.Namespace) }}
```

Comment:
Uhm, this may result in `.Release.Namespace` being duplicated which may cause problems (or inefficiencies). Would you mind changing the type of `NewCluster()`'s `allowedNamespaces` argument and `Cluster.allowedNamespaces` from `[]string` to a set (`map[string]struct{}`)?

Reply:
Good catch. I didn't think of that 👍